### ISSUE TITLE
blockchair.com

### ISSUE THREAD

**Issue by DandelionSprout** [easylist/easylist#9981]
### List the website(s) you're having issues:

`https://blockchair.com/litecoin/transaction/d17d587dad4af55c081fd7b0818556c9a5e8956904b41b087ac37110a8c40d86`

### What happens?

Various ads all over the main view area, half of them for online casinos.

### List Subscriptions you're using:

![image](https://user-images.githubusercontent.com/22780683/145255961-738a26b8-fd53-4c98-9134-63e6aa13f48b.png)

### Your settings

- OS/version: Windows 11 stable x64
- Browser/version: Chrome 96.0.4664.93 x64
- Adblock Extension/version: uBO 1.39.1rc0 + IDCAC 3.3.5

### Other details:

I propose:
```
blockchair.com##.native-sentence
blockchair.com##.top-buttons
blockchair.com##.bottom--buttons-container
```

![image](https://user-images.githubusercontent.com/22780683/145255614-7e5d115b-ea45-42d9-9cb3-72a62ce8fd6d.png)

**Comment by Har01d** [comment]
Hello,

Ads are the only way for Blockchair not to paywall lots of the features we provide for free.

If you constantly use the website and the ads are too distractive, we have an option to turn off all the ads in the top menu:

<img width="275" alt="image" src="https://user-images.githubusercontent.com/14060856/145293754-d5ad6132-f181-43af-a773-203e93fd4527.png">

Once activated you won't see any ads:

<img width="1338" alt="image" src="https://user-images.githubusercontent.com/14060856/145293823-c652399a-e0da-476a-a2eb-03bf41f44118.png">

Please consider closing this.

**Comment by LikesHisComputer** [comment]
```
##[href^="https://campaign.aax.com"]
##[href^="https://refpazkjixes.top"]
||refpazkjixes.top^$document
```
(these 3 filters don't address all the ads I encountered)
@ryanbr

**Comment by DandelionSprout** [comment]
>Please consider closing this.

While I'm generally speaking a compassionate man in the adblock industry, there are three aspects that result in me not closing it unilaterally:

* I had no idea the "Turn off ads for 3 days" option existed (though I applaud that it's available for guest users too).
* I've got a major beef with online casinos for 3 years now.
* As someone who do contract-work for Adblock Plus, I get a couple of Euros for each valid report I submit to EasyList. 😅 

In mildening favour, I admit my entries above were pretty broad in order to be sure I blocked all online casinos, even if it also resulted in blocking less bad ads for coin trading sites. I also personally understand it as such that Acceptable Ads would be free of charge for Blockchair to request the help of, which'd whitelist the ads for one-third of ABP and AdBlock users (but not uBO or AdGuard).

I'll let *ryanbr* get the primary final say on all this.
